mozci/data/base.py

```diff
@@ -69,3 +69,11 @@ def get(self, name: str, **context: Any) -> Dict[Any, Any]:
         # Validate output.
         contract.validate_out(result)
         return result
+
+
+def register_sources():
```

Comment:
Were you thinking of allowing external users to define their own datasource or would they always use the built-in ones?